pilot/cmd/pilot-agent/main.go

```diff
@@ -341,6 +341,20 @@ func waitForCompletion(ctx context.Context, fn func(context.Context)) {
 	wg.Done()
 }
 
+// TODO: move this to API in 1.2
+func handleTDDEnvVar() time.Duration {
```

Comment:
nit: this is a slightly odd name for the function ... maybe just `getTerminationDrainDuration`?

Reply:
I would settle for some comments :-), but the better name is nice too.